### DISCUSSION TITLE
How to set an indoor scenario？

### DISCUSSION THREAD

**Discussion by likehbbfoe** [NVlabs/sionna#498] [Q&A] [answered]
Dear developers:
      Is there any document for setting an  indoor scenario for sionna? Only using the openstreet map is not enough accuracy and can not set the indoor scenario. How can I set up a more accurate scenario and indoor scenario?

**Comment by faycalaa** [comment, marked as answer]
Hi @likehbbfoe ,

You can load any Mitsuba scene in the Sionna ray tracer, including a scene modelling an indoor environment.
As an example, you could have a look at [this repository](https://github.com/NVlabs/diff-rt-calibration) which uses an indoor scene.
